src/ConvNetSharp.Flow/Serialization/Onnx/OnnxLoader.cs

```diff
@@ -156,6 +161,31 @@ private Op<T> GetConvOp(NodeProto node, Dictionary<string, Op<T>> seen)
             return op;
         }
 
+        private Op<T> GetGemmOp(NodeProto node, Dictionary<string, Op<T>> seen)
+        {
+            var input = this.GetOp(node.Inputs[0], seen);
+
+            var weightVolume = BuildVolume(this._initializers[node.Inputs[1]]);
+            var biasVolume = BuildVolume(this._initializers[node.Inputs[2]]);
+            var weight = this._cns.Variable(weightVolume, $"Filter_{weightVolume.Shape.Dimensions[3]}", true);
+            var bias = this._cns.Variable(biasVolume, "Bias", true);
+
+            var data = new Dictionary<string, object>
+            {
+                ["Alpha"] = node.Attributes.First(o => o.Name == "alpha").F, // TODO: use alpha
+                ["Beta"] = node.Attributes.First(o => o.Name == "beta").F, // TODO: use beta
+                // ["TransA"] = node.Attributes.First(o => o.Name == "transA").I.ToString(), // TODO: use transA
```

Comment:
Why is it commented ?

Reply:
Oh! This is important. These parameters aren't always present (in my case only `transA` was missing) so the code should handle checking for their existence. I'm new to C# and I wasn't sure what is the best approach for that...